src/pages/memberspage/MemberCard.tsx

```
@@ -0,0 +1,60 @@
+import { useState } from "react";
+import members, { Role, Member } from "../../store/members";
+import Modal from "./modal.tsx";
+import { roleLabels, roleStyles, TEAM_LEADER_IDS } from "./roles.ts";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)